primitives/runtime-interface/src/lib.rs

```diff
@@ -22,6 +22,8 @@
 //! maps to an external function call. These external functions are exported by the wasm executor
 //! and they map to the same implementation as the native calls.
 //!
+//! On a meta level this implies, the runtime calls the functions provided by the generated code.
```

Comment:
I don't get this comment? On which meta level?

Reply:
Should have written calling direction or birds eye view on the architecture.

Reply:
If you want to keep this, please give the sentence more love. First, I would not talk about "generated code" here. 

Maybe you could update the sentences above where you think information is missing?

Reply:
Removed.